### PR TITLE
Add global footer

### PR DESCRIPTION
**Pull request by vassalloandrea**
Fixes: #18 

- Add the footer colors and dimensions
- Add the `Footer` container
- Add the new components to the storybook

image

**Important: I used the `rem` unit to define dimensions and now they are different from the solidus starter frontend because we didn't set the body `font-size` yet**

### CODE REVIEW

File: components/link/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledLink = styled.a`
+  color: ${(props) => props.theme.color.copy.light};
+`
+
+const Link = ({ children, href }) => (
```
Comment (dinohamzic) [inline components/link/index.js:8]: I'm not sure having a custom <Link> component is necessary. I'd use the Next.js `Link` component directly and I'd set the color globally, or on the parent component. Thoughts?

File: lib/theme.js [hunk 2]
```diff
@@ -1,8 +1,22 @@
+const palette = {
+  white: '#FFFFFF',
+  blue: '#3C76f0',
+  concrete: '#F2F2F2',
+  boulder: '#777777'
+}
+
 export const theme = {
+  dimens: {
```
Comment (dinohamzic) [inline lib/theme.js:9]: Is this a typo?

File: lib/theme.js [hunk 3]
```diff
@@ -1,8 +1,22 @@
+const palette = {
```
Comment (dinohamzic) [inline lib/theme.js:1]: Love this, good idea.

**Review by dinohamzic — CHANGES_REQUESTED** [review]

File: lib/theme.js (hunk 2 above)
Comment (vassalloandrea) [reply lib/theme.js:9]: `dimensions` is too long 😄  but I can use it.

File: components/link/index.js (hunk 1 above)
Comment (vassalloandrea) [reply components/link/index.js:8]: Makes sense, I'll refactor it

**Review by rainerdema — APPROVED** [review]
👏

**Review by dinohamzic — APPROVED** [review]

**Pull request merged**